test/minikube/minikube_base_database_test.go

```diff
@@ -163,71 +164,29 @@ func TestKubernetesBasicDatabase(t *testing.T) {
 	helmChartReleaseName, namespaceName := testlib.StartAdmin(t, &options, 1, "")
 
 	admin0 := fmt.Sprintf("%s-nuodb-cluster0-0", helmChartReleaseName)
-	headlessServiceName := fmt.Sprintf("demo")
-	clusterServiceName := fmt.Sprintf("demo-clusterip")
 
-	t.Run("startDatabaseStatefulSet", func(t *testing.T) {
-		defer testlib.Teardown(testlib.TEARDOWN_DATABASE) // ensure resources allocated in called functions are released when this function exits
-
-		testlib.StartDatabase(t, namespaceName, admin0, &helm.Options{
-			SetValues: map[string]string{
-				"database.sm.resources.requests.cpu":    testlib.MINIMAL_VIABLE_ENGINE_CPU,
-				"database.sm.resources.requests.memory": testlib.MINIMAL_VIABLE_ENGINE_MEMORY,
-				"database.te.resources.requests.cpu":    testlib.MINIMAL_VIABLE_ENGINE_CPU,
-				"database.te.resources.requests.memory": testlib.MINIMAL_VIABLE_ENGINE_MEMORY,
-				"database.te.labels.cloud":              LABEL_CLOUD,
-				"database.te.labels.region":             LABEL_REGION,
-				"database.te.labels.zone":               LABEL_ZONE,
-				"database.sm.labels.cloud":              LABEL_CLOUD,
-				"database.sm.labels.region":             LABEL_REGION,
-				"database.sm.labels.zone":               LABEL_ZONE,
-			},
-		})
+	defer testlib.Teardown(testlib.TEARDOWN_DATABASE) // ensure resources allocated in called functions are released when this function exits
 
-		t.Run("verifySecret", func(t *testing.T) { verifySecret(t, namespaceName) })
-		t.Run("verifyDBHeadlessService", func(t *testing.T) { verifyDBService(t, namespaceName, admin0, headlessServiceName, true) })
-		t.Run("verifyDBClusterService", func(t *testing.T) { verifyDBService(t, namespaceName, admin0, clusterServiceName, false) })
-		t.Run("verifyNuoSQL", func(t *testing.T) { verifyNuoSQL(t, namespaceName, admin0, "demo") })
-		t.Run("verifyPodLabeling", func(t *testing.T) { verifyPodLabeling(t, namespaceName, admin0) })
+	databaseReleaseName := testlib.StartDatabase(t, namespaceName, admin0, &helm.Options{
+		SetValues: map[string]string{
+			"database.sm.resources.requests.cpu":    testlib.MINIMAL_VIABLE_ENGINE_CPU,
+			"database.sm.resources.requests.memory": testlib.MINIMAL_VIABLE_ENGINE_MEMORY,
+			"database.te.resources.requests.cpu":    testlib.MINIMAL_VIABLE_ENGINE_CPU,
+			"database.te.resources.requests.memory": testlib.MINIMAL_VIABLE_ENGINE_MEMORY,
+			"database.te.labels.cloud":              LABEL_CLOUD,
+			"database.te.labels.region":             LABEL_REGION,
+			"database.te.labels.zone":               LABEL_ZONE,
+			"database.sm.labels.cloud":              LABEL_CLOUD,
+			"database.sm.labels.region":             LABEL_REGION,
+			"database.sm.labels.zone":               LABEL_ZONE,
+		},
```

Comment:
The `startDatabaseStatefulSet` subtest is unchanged but only made part of the main test. The `startDatabaseStatefulSetMultiTenant` has been extracted to separate test file.
Maybe it's easy to review this file side-by-side.